### ISSUE TITLE
Add support for @docusaurus/core 3.0.0

### ISSUE THREAD

**Issue by renovate-changelog-viewer** [typesense/docusaurus-theme-search-typesense#38]
## Description

This library throws an error when updating to `@docusaurus/core` version `3.0.0`.

## Steps to reproduce

Upgrade to `@docusaurus/core@3.0.0` in any repository that uses `docusaurus-theme-search-typesense`.

## Expected Behavior

No errors on package installation

## Actual Behavior

error thrown

```
npm install

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: mypackage@0.0.0
npm ERR! Found: @docusaurus/core@3.0.0
npm ERR! node_modules/@docusaurus/core
npm ERR!   @docusaurus/core@"3.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @docusaurus/core@"2.4.3" from docusaurus-theme-search-typesense@0.13.1
npm ERR! node_modules/docusaurus-theme-search-typesense
npm ERR!   dev docusaurus-theme-search-typesense@"^0.13.1" from the root project
```

## Metadata

**Typesense Version**: `1.5.3`

**OS**: mac (intel)

**Comment by jasonbosco** [comment]
Just published `0.14.0-0` which adds support for docusuaurus@3. Could you give it a shot now?

**Comment by jasonbosco** [comment]
Thank you for confirming @neoakris. I just published `0.14.0`.
